### ISSUE TITLE
[ERROR] [nordnm.networkmanager:230]: invalid interpolation syntax in '...' at position ...

### ISSUE THREAD

**Issue by peteruithoven** [Chadsr/NordVPN-NetworkManager#35]
As mentioned in #29 I wasn't able to save my credentials, with 0.0a12 this is now solved, but when I then try to use nordnm it seems to run into a similar issue. I've replaced my password and the position with `...`.

```
$ sudo nordnm -ska nl normal tcp
     _   _               _ _   _ ___  ___
    | \ | |             | | \ | ||  \/  |
    |  \| | ___  _ __ __| |  \| || .  . |
    | . ` |/ _ \| '__/ _` | . ` || |\/| |
    | |\  | (_) | | | (_| | |\  || |  | |
    \_| \_/\___/|_|  \__,_\_| \_/\_|  |_/   v0.0a12 (Latest)

[INFO] [nordnm.nordnm:307]: Checking for new connections to import...
[WARNING] [nordnm.nordnm:327]: Active VPNs and/or kill-switch disabled for accurate benchmarking. Your connection is not secure until these are re-enabled.
[INFO] [nordnm.nordnm:329]: Benchmarking servers...
[INFO] [nordnm.nordnm:337]: Benchmarking complete. Took 13.58 seconds.
[INFO] [nordnm.nordnm:201]: Auto-Connect file removed.
[INFO] [nordnm.nordnm:209]: Removing all active connections...
[ERROR] [nordnm.networkmanager:230]: invalid interpolation syntax in '...' at position ...
[INFO] [nordnm.nordnm:368]: No new connections added.
[WARNING] [nordnm.nordnm:194]: No active server found matching (nl, normal, tcp). Check your input and try again.
[INFO] [nordnm.nordnm:145]: Network kill-switch set successfully!
```

Please let me know if you need more information.

**Comment by Chadsr** [comment]
And this is what happens when we don't do testing thoroughly :)

I indeed did miss more occurrences of this problem. Apologies. I haven't had much free time to put into this project the last weeks.

You'll be glad to know I plan on making some basic test units soon, so hopefully more of this sort of bug will be dealt with before releases...

I will go ahead and release 'hopefully' the final fix. (#36)

**Comment by peteruithoven** [comment]
No apologies needed, I'm happy your putting in the time to make using this vpn easier to use for all of us.

**Comment by Chadsr** [comment]
I just released v0.0a13 with the speed fix. Let me know if you still encounter problems.

**Comment by peteruithoven** [comment]
Works, thanks!
